docs/scenarios/stacks.feature

```diff
@@ -0,0 +1,138 @@
+Feature: As a library member of staff I want to login so I can place a request
+  # Note: users can currently login with either their Wellcome Trust account email address, library card number or library username
+
+  Scenario: User supplies correct user name and password
+
+    Given that I am on the login page
+    When I enter my user name and password correctly and click ‘Login’
+    Then I am taken to the xxxx page
+
+  Scenario: User does NOT enter correct user name and password
+
+    Given that I am on the login page
+    When I enter my user name and password incorrectly and click ‘Login’
+    Then I see an error message ‘incorrect username or password’
+
+
+Feature: As a library member of staff I want to know if an item is available for requesting
+
+  Scenario: A user searches for a work and it’s available to request
+
+    Given that I have searched for an item
+    When I click onto the work page I’m interested in
+    Then I can see the work is available to request
+    And I can see the location of the item
+    And I can see a ‘request it’ button
+
+
+  Scenario: A user searches for a work but it’s NOT available to request
+    Given that I have searched for an item
+    When I click on the work page I’m interested in
+    Then I can see that the work isn’t available to request
+    And I can see an error message “this item is not currently available to request”
+
```

Comment:
I would suggest these two 'unavailable' scenarios: 

`Scenario: A user searches for a work but it's NOT CURRENTLY available to request
`
`Scenario: A users searches for a work but it's NEVER available to request`